src/components/RequestForm/RequestForm.tsx

```diff
@@ -48,7 +49,7 @@ export const RequestForm: React.FunctionComponent<IRequestFormProps> = (props) =
                     // copy that doesn't get changed to pass to the validator
                     setOldRequest(newRequest);
                 } else {
-                    history.push("/Requests");
+                    redirect("/Requests");
```

Comment:
You'll see this comment elsewhere. Same reason (remove failed item from history list)
history.replace("/Requests");